source/02_algo/exploration_rl_en.rst

```diff
@@ -0,0 +1,241 @@
+
+Intrinsic Rewards Based on Information Theory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To encourage exploration, another idea is to design intrinsic rewards based on information theory.
+The paper [11]_ introduced Variational information maximizing exploration (VIME), the core idea is the maximization of information gain about the agent's belief of environment dynamics, using variational inference in Bayesian neural networks, which can efficiently handle continuous state and action spaces.
+The paper [12]_ proposes the EMI algorithm (Exploration with Mutual Information), which does not learn representations through the usual encoding/decoding raw state or action space, but learns the relationship between states and actions by maximizing the mutual information between related state-action representations. sign,
+They experimentally verified that the forward prediction signal extracted in such a representation space can guide exploration well.
+In addition, there are also methods such as DIYAN [13]_, which is based on the objective function of mutual information to learn skill variables, which can automatically learn the distribution of state and skill by setting intrinsic rewards related to mutual information without external rewards. In subsequent hierarchical learning, imitation learning and exploration tasks.
+
+Memory-Based Exploration
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Intrinsic reward-based exploration methods such as ICM and RND propose to measure the novelty of a state by predicting the error of the problem, providing a large intrinsic reward for a state with a large novelty and promoting exploration. These methods are difficult to explore under many sparse reward settings. Good results have been achieved on the task of , but there is a problem: \ **As the number of training steps of the agent increases, the prediction error of the prediction problem begins to decrease, and the exploration signal becomes smaller, that is, the agent is no longer encouraged to visit a certain Some states, but it is possible that these states are the states that must be visited to obtain extrinsic rewards**\ . And there may also be the following problems:
```

Comment:
Doesn't match the original text: "These methods are difficult to explore under many sparse reward settings. Good results have been achieved on the task of , but there is a problem"
 Delete "are the states that" in "that these states are the states that must be visited to obtain extrinsic rewards"